packages/KInput/KInput.vue

```diff
@@ -2,24 +2,35 @@
   <div class="k-input-wrapper">
     <input
       v-if="!label"
```

Comment:
every input should have a label. It can be visually hidden for style purposes

Reply:
Will not address as we doc correct usage of `KLabel` with `KInput`.

![image](https://user-images.githubusercontent.com/67973710/134971930-4084e16c-058f-4f20-9878-579e9d4db995.png)